src/ledger/tests.zig

```diff
@@ -330,12 +330,18 @@ pub const TestState = struct {
 };
 
 pub const TestDB = struct {
-    const dir = sig.TEST_STATE_DIR ++ "/blockstore";
+    const dir = sig.TEST_STATE_DIR ++ "blockstore";
 
     pub fn init(comptime test_src: std.builtin.SourceLocation) !BlockstoreDB {
         return try initCustom(std.testing.allocator, test_src);
     }
 
+    pub fn reuseBlockstore(comptime test_src: std.builtin.SourceLocation) !BlockstoreDB {
+        const path = comptimePrint("{s}/{s}/{s}", .{ dir, test_src.file, test_src.fn_name });
+        try std.fs.cwd().makePath(path);
```

Comment:
If this reuses an existing blockstore, I'd expect this to return `error.PathAlreadyExists`. Are you expecting this folder to exist?

Reply:
> I'd expect this to return error.PathAlreadyExists

That is actually not the documented behaviour. As can be seen here:

https://github.com/ziglang/zig/blob/87863a834b074311d70ec4dd3b10c74705fff54d/lib/std/fs/Dir.zig#L1142

It would return success if the path already exists and is a director.

> Are you expecting this folder to exist?

Yes. Maybe I rename it to `reusePath`?

Reply:
my mistake. seems fine as is